neo/mmakefile.src

```diff
@@ -0,0 +1,50 @@
+# Copyright � 2012-2017, Nick "Kalamatee" Andrews.
```

Comment:
again: could this be in `neo/sys/aros/` instead?

Of that's really not possible, at least mention somewhere at the very beginning in a comment that this is for aros,
so people don't mistake it for a generic makefile